pages/index.js

```diff
@@ -141,11 +167,16 @@ const Home = ({ latestStories }) => {
 
   const [caseData, setCaseData] = useState({})
   const [cumulativeCase, setCumulativeCase] = useState(null)
+  const [cumulativeTest, setCumulativeTest] = useState(null)
 
   const [cumulativeData, setCumulativeData] = useState({})
   const [totalCases, setTotalCases] = useState(null)
   const [totalCasesDate, setTotalCasesDate] = useState(null)
   
+  const commaNumber = (number) => {
+    if (number == null) {return "0"}
```

Comment:
prefer === or even better just check if number is falsy. Also could make this function a single liner with a ternary operator

Reply:
when you do that you can even ditch the brackets for true functional programming goodness